about/list_of_features.rst

```diff
@@ -32,7 +32,6 @@ Platforms
 
 - Android 4.4 and later.
 - iOS 10.0 and later.
-- HTML5 via WebAssembly (Firefox, Chrome, Edge, Opera).
```

Comment:
HTML5 support is no longer listed due to the lack of a *production-ready* WebGL backend as of writing. It can be listed again once such a renderer is available (at least for 2D).